src/components/Filters/components/Input/Input.tsx

```diff
@@ -52,17 +63,23 @@ const Input: React.FC<InputProps> = ({
   maxLength,
   pattern,
   patternMessage,
+  onValidate = validatePattern,
   placeholder = 'String',
+  isDisabled = false,
+  onError,
 }) => {
-  const handleOnChange = (event) => {
-    if (patternMessage) {
-      const { target } = event;
-      target.setCustomValidity(
-        target.validity.patternMismatch ? patternMessage : '',
-      );
-    }
+  const [isInvalid, setIsInvalid] = useState(false);
+
+  const handleOnValidate = (event) => {
+    const { target } = event;
+    const error = onValidate(target);
```

Comment:
Same here, `onValidate` should return boolean and name should have boolean prefix

Reply:
I updated the names for all inputs but in the Input and TagsInput case we need the whole target because 
```
validatePattern = (target: HTMLInputElement): boolean => {
  return !!target.validity.patternMismatch;
};```

Reply:
Okey. `onValidate` is not custom validation here, it mislead me a bit.